src/StackExchange.Utils.Http/Http.cs

```diff
@@ -108,5 +108,15 @@ internal static async Task<HttpCallResponse<T>> SendAsync<T>(IRequestBuilder<T>
 
             return result;
         }
+
+        private const string Switch_AllowUnencryptedHttp2 = "System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport";
+        /// <summary>
+        /// Allows HttpClient to use HTTP/2 without TLS
+        /// </summary>
+        public static bool AllowUnencryptedHttp2
+        {
+            get => AppContext.TryGetSwitch(Switch_AllowUnencryptedHttp2, out var enabled) && enabled;
+            set => AppContext.SetSwitch(Switch_AllowUnencryptedHttp2, value);
+        }
```

Comment:
Definitely not a fan of the static, at least on this class (it's on the default intellisense for every intended usage, etc.) Perhaps on `HttpSettings` and `GloballyAllow...`?

Reply:
It is fundamentally a global thing; static on `HttpSettings` work for you?